Corona-Warn-App/src/main/java/de/rki/coronawarnapp/diagnosiskeys/download/DownloadDiagnosisKeysTask.kt

```diff
@@ -70,18 +73,30 @@ class DownloadDiagnosisKeysTask @Inject constructor(
             checkCancel()
 
             // RETRIEVE RISK SCORE PARAMETERS
-            val exposureConfiguration = appConfigProvider.getAppConfig().exposureDetectionConfiguration
+            val exposureConfig: ExposureDetectionConfig = appConfigProvider.getAppConfig()
 
             internalProgress.send(Progress.ApiSubmissionStarted)
             internalProgress.send(Progress.KeyFilesDownloadStarted)
 
             val requestedCountries = arguments.requestedCountries
-            val availableKeyFiles = getAvailableKeyFiles(requestedCountries)
+            val keySyncResult = getAvailableKeyFiles(requestedCountries)
             checkCancel()
 
-            val totalFileSize = availableKeyFiles.fold(0L, { acc, file ->
-                file.length() + acc
-            })
+            val trackedDetections = enfClient.latestCalculations().first()
```

Comment:
why can there be multiple "latest calculations"? is this really "calculations" or "exposure detections"?

Reply:
> why can there be multiple "latest calculations"?

It's a `Flow`, it will emit multiple values over time,  but always the latest.

>  is this really "calculations" or "exposure detections"?

Calculation by the ENF.